R/functions.R

```diff
@@ -658,6 +658,26 @@ setMethod("kronecker", signature(X = "greta_array", Y = "greta_array"),
           }
 )
 
+#' @import methods
+setMethod(kronecker, signature(X = "array", Y = "greta_array"),
+            function(X, Y, FUN = c("*", "/", "+", "-"), make.dimnames = FALSE,
+                   ...) {
+
+              kronecker(as.greta_array(X), Y, FUN, make.dimnames = FALSE)
```

Comment:
This looks great. I'm not sure whether another call to `kronecker`, or `callNextMethod()` is the idiomatic thing here, but I'm happy with this if it works

Reply:
I don't think `callNextMethod` is appropriate here as we're not operating on a object with multiple classes  - we're coercing and then calling a method. (I'm certainly not an expert on S4 though)

Reply:
OK, great!